src/api-docs/source/index.html.md

```diff
@@ -109,7 +109,18 @@ Note that if multiple user-agents are using the wallet at the same time,
 the wallet RPC API is not safe as it maintains global state (more on this below)
 It is recommended to use the wallet HTTP API in multiprocess/multithreaded environments.
 
-**Indexing**
+## Values
+
+Handshake currency HNS has smallest unit — one millionth of the whole, called dollarydoo. 
+<br>So <code>1 HNS = 1 000 000 dollarydoos</code>.
+<br>Different APIs require <code>value</code> and <code>rate</code> parameters nominated either in dollarydoos
+(subunits) which are integer values or in the whole HNS, which are floats.
+
+<code>value</code> and <code>rate</code> are expressed in dollarydoos when using cURL or Javascript.<br>
+<code>value</code> and <code>rate</code> are expressed in whole HNS when using CLI and RPC.
```

Comment:
Great approach with the anchored links! Just to be nitpicky about the English here, can we try:


> The smallest unit of the Handshake currency (HNS) is called the "dolalrydoo" — one millionth of the whole.
> So 1 HNS = 1,000,000 dollarydoos.
> Different APIs require value and fee rate parameters denominated either in dollarydoos (subunits) which are integer values or in whole HNS, which are floats.
> 
> When using HTTP API endpoints:
> Value and rate are expressed in dollarydoos when using cURL or Javascript.
> Value and rate are expressed in whole HNS when using CLI commands.
> 
> When using RPC API commands:
> Value and rate are always expressed in whole HNS, no matter the entry method.

(and please double check that I'm correct, pretty sure that RPC always uses HNS no matter if its CLI or cURL or JS.